components/courses/app/models/courses/lecture.rb

```diff
@@ -17,5 +17,11 @@ class Lecture < ApplicationRecord
     enum status: [UPCOMING, PASSED]
 
     validates :title, :started_at, :finished_at, :venue_id, :mentor_id, presence: true
+    validate  :time_policy
```

Comment:
Well, that was the best name I could find at the moment. It checks that finish time is not before the start time, so it was logical for me to name it like this. If you have better name, please tell me.